domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -2611,20 +2613,17 @@ public ResultsHelper(SQLRowStream sqlRowStream, Class<T> clazz) {
    * @param o
    * @param row
    */
-  void populateExternalColumns(Map<String, Method> externalColumnSetters, Object o, JsonObject row) {
+  void populateExternalColumns(Map<String, Method> externalColumnSetters, Object o, Row row)
+      throws InvocationTargetException, IllegalAccessException {
     for (Map.Entry<String, Method> entry : externalColumnSetters.entrySet()) {
       String columnName = entry.getKey();
       Method method = entry.getValue();
```

Comment:
```suggestion
    externalColumnSetters.forEach((columnName, method) -> {
```
and properly indent the following lines.

Reply:
May throw exception, so I'm skipping that for now